src/cclib/method/lpa.py

```diff
@@ -16,8 +16,17 @@
 
 class LPA(Population):
     """The Löwdin population analysis"""
+
     def __init__(self, *args):
 
+        self.required_attrs = ('mocoeffs','homos','nbasis')
+
+        # If the parsed data does NOT have either of the Fragment orbital overlap 
+        # or the Atomic orbital overlap matrix
+        if not (hasattr(data, "aooverlaps") \
+             or hasattr(data, "fooverlaps") ):
+            self.required_attrs += ('aooverlaps or fooverlaps',)

```

Comment:
I don't think this will work here or in the other parsers. Look at https://github.com/cclib/cclib/blob/829c0ec9ed1ca1cc335575440a7ee0ae926d5e1d/cclib/method/calculationmethod.py#L53.

Reply:
Done in #642.